pom.xml

```diff
@@ -559,22 +559,9 @@
                   <extraArtifacts>
                     <extraArtifact>${project.groupId}:${project.artifactId}:${project.version}:license.properties:test-third-party</extraArtifact>
                   </extraArtifacts>
```

Comment:
I guess the changes in this file made sense while you were testing your work, but actually should not have been committed, right?

Reply:
yes that it's an error. The PR would just demostrate the issue. We did work to a solution, we did a workaround for the release. A fix could be simply skip projects if are part of the reactor (not tested).

Reply:
Could you please describe your workaround in https://github.com/mojohaus/license-maven-plugin/issues/317 for other people who will hit the same issue in the future?

Reply:
Change the preparationGoals of the mave-release-plugin from it default value (clean verify) to clean install so that when the release plugin run the second phase (release) deploy site:deploy than the site generation of the parent project has jars of submodules available in the local m2.